traffic_ops/traffic_ops_golang/crconfig/deliveryservice.go

```diff
@@ -308,7 +308,7 @@ and d.active = true
 			}
 		}
 
-		if consistentHashRegex.String != "" {
+		if consistentHashRegex.Valid && consistentHashRegex.String != "" {
```

Comment:
This is actually better than what I suggested (assuming we don't want to allow empty strings for the consistent hash regular expression), just wanted to point that out.